README.md

```diff
@@ -12,7 +12,7 @@ Run the container and publish port 25565. You can mount `/srv/minecraft` to a vo
 $ docker volume create mc-eternal-vol
 mc-eternal-vol
 # Create container and mount Minecraft data to volume
-$ docker run -v mc-eternal-vol:/srv/minecraft -p 25565:25565 -i -d mymindstorm/mc-eternal:1.3.3
+$ docker run -v mc-eternal-vol:/srv/minecraft -p 25566:25565 -p 25576:25575 -p 8124:8123 -i -d --name mceternal tysseract/mc-eternal:1.3.6
```

Comment:
`tysseract/mc-eternal` is not the name of the docker container that will be published. Please revert this.

Reply:
I'm not very good at GitHub I didn't mean for this to get committed...